docs/pages/eas/ai/mcp.mdx

```diff
@@ -0,0 +1,185 @@
+3. Select **HTTP**
+4. Enter the server details:
+   - **URL**: `https://mcp.expo.dev/mcp`
+   - **Name**: expo-mcp
+
+</Collapsible>
+
+</Step>
+
+<Step label="2">
+
+### Authenticate with Expo
+
+After installing the MCP server, you'll need to authenticate using one of two methods:
+
+#### Access token (recommended)
+
+Generate an access token from your Expo account settings and use it during the OAuth flow.
+
+#### Credentials
+
+Use your Expo account username and password. In this case, the server will generate an access token automatically.
+
+</Step>
+
+<Step label="3">
+
+### Set up local capabilities (Recommended)
+
+> **info** Local capabilities are only available in **SDK 54 and later**.
+
+For the full MCP experience with advanced features like taking screenshots from your iOS Simulator, opening DevTools, and automation capabilities, set up a local Expo development server:
+
+<Terminal
+  cmd={[
+    '$ cd /path/to/your-project',
+    '',
+    '# Install expo-mcp package',
+    '$ npx expo install expo-mcp --dev',
+    '',
+    '# Ensure you are logged in from CLI with the same account as the one used to authenticate with MCP server',
+    '$ npx expo whoami || npx expo login',
+    '',
+    '# Start dev server with MCP capabilities',
+    '$ EXPO_UNSTABLE_MCP_SERVER=1 npx expo start',
+  ]}
+/>
+
+> **important** Whenever you start or stop the development server, you need to **reconnect or restart** your MCP server connection in your AI tool to ensure the AI tool gets refreshed capabilities.
+
+</Step>
+
+## Server capabilities vs local capabilities
+
+The Expo MCP server provides two types of capabilities depending on your setup:
+
+### Server capabilities
+
+Server capabilities are available with just the remote MCP server connection, without needing to set up a local development server. The **generate_agents_md** tool is an example of a server capability.
+
+### Local capabilities
+
+Local capabilities require a local Expo development server to be running and provide advanced features that interact with your local development environment:
+
+- **Automation tools**: Take screenshots, tap views, find elements by testID
+- **Development tools**: Open React Native DevTools
+- **Project analysis**: Generate `expo-router` sitemap
+
+These capabilities enable more sophisticated workflows like automated testing, visual verification, and deeper project introspection. To use local capabilities, you will need to follow the [Set up local capabilities](#set-up-local-capabilities-recommended) section above.
+
+## Available MCP capabilities
+
+> **info** The MCP capabilities are subject to change from the `expo-mcp` package updates or MCP server changes. The follow list is just a reference and may not be up to date.
+
+### Tools
+
+| Tool                                   | Description                                                          | Example Prompt                                      | Availability                           |
```

Comment:
Should `search_documentation` and `learn` be a part of this table? They are enabled by default:

<img width="958" height="338" alt="CleanShot 2025-10-02 at 16 52 53@2x" src="https://github.com/user-attachments/assets/bba1e211-24e3-4f3e-aff1-cb5b9ed14edf" />

The reason I am asking this is to have an example prompt below in the able. It will be nice to have.

Reply:
added